build/vnu-jar.js

```diff
@@ -14,7 +14,7 @@ const vnu = require('vnu-jar')
 
 childProcess.exec('java -version', (error, stdout, stderr) => {
   if (error) {
-    console.error('Skipping vnu-jar test; Java is missing.')
```

Comment:
it seems more appropriate to use `console.error` here because the line before you have `if (error)`

Reply:
Yeah, I know, I'm in the middle too. I will probably revert this.